languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/multiwords.txt

```diff
@@ -1516,4 +1516,2205 @@ Luperque; N m s
+Les Ancizes-Comps;Z e sp
+Les Angles-sur-Corrèze;Z e sp
+Les Anses-d'Arlet;Z e sp
+Les Artigues-de-Lussac;Z e sp
+Les Autels-Villevillon;Z e sp
+Les Authieux-du-Puits;Z e sp
+Les Authieux-sur-Calonne;Z e sp
+Les Authieux-sur-le-Port-Saint-Ouen;Z e sp
+Les Avanchers-Valmorel;Z e sp
+Les Avenières Veyrins-Thuellin;Z e sp
+Les Baux-Sainte-Croix;Z e sp
+Les Baux-de-Breteuil;Z e sp
+Les Baux-de-Provence;Z e sp
+Les Bordes-Aumont;Z e sp
+Les Bordes-sur-Arize;Z e sp
+Les Cent-Acres;Z e sp
+Les Champs-Géraux;Z e sp
+Les Chapelles-Bourbon;Z e sp
+Les Chavannes-en-Maurienne;Z e sp
+Les Châtelliers-Notre-Dame;Z e sp
+Les Clayes-sous-Bois;Z e sp
+Les Contamines-Montjoie;Z e sp
+Les Corvées-les-Yys;Z e sp
+Les Costes-Gozon;Z e sp
+Les Côtes-d'Arey;Z e sp
+Les Côtes-de-Corps;Z e sp
+Les Deux-Fays;Z e sp
+Les Deux-Villes;Z e sp
+Les Essards-Taignevaux;Z e sp
+Les Essarts-le-Roi;Z e sp
+Les Essarts-le-Vicomte;Z e sp
+Les Essarts-lès-Sézanne;Z e sp
+Les Grandes-Armoises;Z e sp
+Les Grandes-Chapelles;Z e sp
+Les Grandes-Loges;Z e sp
+Les Grandes-Ventes;Z e sp
+Les Grands-Chézeaux;Z e sp
+Les Granges-Gontardes;Z e sp
+Les Granges-le-Roi;Z e sp
+Les Hautes-Rivières;Z e sp
+Les Hauts-Talican;Z e sp
+Les Hauts-d'Anjou;Z e sp
+Les Hauts-de-Caux;Z e sp
+Les Hauts-de-Chée;Z e sp
+Les Hôpitaux-Neufs;Z e sp
+Les Hôpitaux-Vieux;Z e sp
+Les Isles-Bardel;Z e sp
+Les Istres-et-Bury;Z e sp
+Les Landes-Genusson;Z e sp
+Les Loges-Marchis;Z e sp
+Les Loges-Margueron;Z e sp
+Les Loges-Saulces;Z e sp
+Les Loges-en-Josas;Z e sp
+Les Loges-sur-Brécey;Z e sp
+Les Lucs-sur-Boulogne;Z e sp
+Les Lèves-et-Thoumeyragues;Z e sp
+Les Magnils-Reigniers;Z e sp
+Les Martres-d'Artière;Z e sp
+Les Martres-de-Veyre;Z e sp
+Les Moitiers-d'Allonne;Z e sp
+Les Monts-Ronds;Z e sp
+Les Monts-Verts;Z e sp
+Les Moutiers-en-Auge;Z e sp
+Les Moutiers-en-Cinglais;Z e sp
+Les Moutiers-en-Retz;Z e sp
+Les Noës-près-Troyes;Z e sp
+Les Ollières-sur-Eyrieux;Z e sp
+Les Ormes-sur-Voulzie;Z e sp
+Les Pavillons-sous-Bois;Z e sp
+Les Pennes-Mirabeau;Z e sp
+Les Petites-Armoises;Z e sp
+Les Petites-Loges;Z e sp
+Les Plains-et-Grands-Essarts;Z e sp
+Les Planches-en-Montagne;Z e sp
+Les Planches-près-Arbois;Z e sp
+Les Ponts-de-Cé;Z e sp
+Les Portes-en-Ré;Z e sp
+Les Rivières-Henruel;Z e sp
+Les Roches-de-Condrieu;Z e sp
+Les Roches-l'Évêque;Z e sp
+Les Rouges-Eaux;Z e sp
+Les Rues-des-Vignes;Z e sp
+Les Sables-d'Olonne;Z e sp
+Les Salles-Lavauguyon;Z e sp
+Les Salles-de-Castillon;Z e sp
+Les Salles-du-Gardon;Z e sp
+Les Salles-sur-Verdon;Z e sp
+Les Souhesmes-Rampont;Z e sp
+Les Terres-de-Chaux;Z e sp
+Les Thilliers-en-Vexin;Z e sp
+Les Touches-de-Périgny;Z e sp
+Les Trois-Bassins;Z e sp
+Les Trois-Châteaux;Z e sp
+Les Trois-Domaines;Z e sp
+Les Trois-Moutiers;Z e sp
+Les Trois-Pierres;Z e sp
+Les Trois-Îlets;Z e sp
+Les Velluire-sur-Vendée;Z e sp
+Les Ventes-de-Bourse;Z e sp
+Les Villards-sur-Thônes;Z e sp
+Les Églises-d'Argenteuil;Z e sp
+Les Églisottes-et-Chalaures;Z e sp
+Morannes sur Sarthe-Daumeray;Z e sp
```

Comment:
In multiwords with determiner (Le, La, Les...) we can specify gender and number. For example, La -> Z f s